tests/integration/docusaurus/validation/checkpoints/how_to_pass_an_in_memory_dataframe_to_a_checkpoint.py

```diff
@@ -1,24 +1,22 @@
 # <snippet name="tests/integration/docusaurus/validation/checkpoints/how_to_pass_an_in_memory_dataframe_to_a_checkpoint.py setup">
 import pandas
 import great_expectations as gx
-from great_expectations.checkpoint import SimpleCheckpoint
 
 context = gx.get_context()
 # </snippet>
 
-context.add_or_update_expectation_suite("my_expectation_suite")
+# context.add_or_update_expectation_suite("my_expectation_suite")
```

Comment:
@donaldheppner Commenting out code in documentation does not seem appropriate.  Are you certain about this?  Thanks.

Reply:
ty! fixed! (I removed the commented out code, it is not included in a snippet)